### PR TITLE
Adding a project to generate sql p fake data

### PR DESCRIPTION
**Pull request by WojcikMike**

### CODE REVIEW

File: src/TimeoutMigrationTool.SqlP.FakeData/TimeoutMigrationTool.SqlP.FakeData.csproj [hunk 1]
```diff
@@ -0,0 +1,85 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (andreasohlund) [inline src/TimeoutMigrationTool.SqlP.FakeData/TimeoutMigrationTool.SqlP.FakeData.csproj:1]: This is the old csproj format, copy the CS proj from another project?

File: src/TimeoutMigrationTool.SqlP.FakeData/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
```
Comment (andreasohlund) [inline src/TimeoutMigrationTool.SqlP.FakeData/Properties/AssemblyInfo.cs:1]: This entire file can be removed

Comment (WojcikMike) [reply src/TimeoutMigrationTool.SqlP.FakeData/Properties/AssemblyInfo.cs]: Done

File: src/TimeoutMigrationTool.SqlP.FakeData/TimeoutMigrationTool.SqlP.FakeData.csproj (hunk 1 above)
Comment (WojcikMike) [reply src/TimeoutMigrationTool.SqlP.FakeData/TimeoutMigrationTool.SqlP.FakeData.csproj]: Done

File: src/TimeoutMigrationTool.SqlP.FakeData/TimeoutMigrationTool.SqlP.FakeData.csproj [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.3</LangVersion>
+    <TargetFramework>net472</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="NServiceBus.Persistence.Sql" Version="5.*" />
```
Comment (andreasohlund) [inline src/TimeoutMigrationTool.SqlP.FakeData/TimeoutMigrationTool.SqlP.FakeData.csproj:8]: We don't use wildcards to get deterministic builds

**Pull request merged**